tox fails on pendulum.tz

Seeing the following error lately:

```
.tox/py38-unit/lib/python3.8/site-packages/airflow/settings.py:51: in <module>
    TIMEZONE = pendulum.tz.timezone("UTC")
E   TypeError: 'module' object is not callable
```

--- a/dags/openshift_nightlies/manifest.yaml
+++ b/dags/openshift_nightlies/manifest.yaml
@@ -1,16 +1,16 @@
 versions:
-  - version: "4.12"
+  - version: "4.15"
     alias: stable
-    releaseStream: 4.12.0-0.nightly
-    baremetalReleaseStream: latest-4.12
-  - version: "4.13"
+    releaseStream: 4.15.0-0.nightly
+    baremetalReleaseStream: latest-4.15
+  - version: "4.16"
     alias: next
-    releaseStream: 4.13.0-0.nightly
-    baremetalReleaseStream: latest-4.13
-  - version: "4.14"
+    releaseStream: 4.16.0-0.nightly
+    baremetalReleaseStream: latest-4.16
+  - version: "4.17"
     alias: future
-    releaseStream: 4.14.0-0.nightly
-    baremetalReleaseStream: latest-4.14
+    releaseStream: 4.17.0-0.nightly
+    baremetalReleaseStream: latest-4.17
 
 
 dagConfig:
@@ -29,7 +29,7 @@ dagConfig:
 
 platforms:
   cloud:
-    versions: ["4.12", "4.13", "4.14"]
+    versions: ["4.15", "4.16", "4.17"]
     providers: ["aws"]
     #providers: ["aws", "aws-arm", "azure","gcp"]
     variants:
@@ -76,7 +76,7 @@ platforms:
 
   # Do not program concurrent builds of ROSA/ROGCP/ARO
   rosa:
-    versions: ["4.12", "4.13"]
+    versions: ["4.15", "4.16"]
     variants:
       - name: sts-ovn-small-cp
         schedule: "0 12 * * 3"
@@ -115,7 +115,7 @@ platforms:
           benchmarks: upgrade.json
 
   rosahcp:
-    versions: ["4.12", "4.13"]
+    versions: ["4.15", "4.16"]
     variants:
       - name: p75-control-plane
         schedule:  "0 12 * * 3"
@@ -134,7 +134,7 @@ platforms:
           benchmarks: hcp-small-control-plane.json
 
   rogcp:
-    versions: ["4.12", "4.13"]
+    versions: ["4.15", "4.16"]
     variants:
       - name: ovn-control-plane
         schedule: "10 13 * * 1"

--- a/dags/requirements.txt
+++ b/dags/requirements.txt
@@ -4,3 +4,4 @@ prometheus-api-client==0.5.2
 elasticsearch==7.13.4
 apache-airflow-providers-slack==7.3.0
 pydantic>=1.10.0,<2.0.0 # https://github.com/apache/airflow/issues/32311
+pendulum<3.0.0

--- a/dags/tests/openshift_nightlies/util/test_manifest.py
+++ b/dags/tests/openshift_nightlies/util/test_manifest.py
@@ -7,10 +7,10 @@ from os import environ
 from unittest.mock import patch
 
 class TestManifest():
-    release_stream_base_url = "https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/api/v1/releasestream"
-    stream = "4.12.0-0.nightly"
-    versions = { "4.12" }
-    year = "2023"
+    release_stream_base_url = "https://amd64.ocp.releases.ci.openshift.org/api/v1/releasestream"
+    stream = "4.17.0-0.nightly"
+    versions = { "4.17" }
+    year = "2024"
     INSTALL_BINARY="openshift_install_binary_url"
     CLIENT_BINARY="openshift_client_location"
 
@@ -30,13 +30,13 @@ class TestManifest():
 
     def assert_amd_installer(self,stream):
         assert "arm64" not in stream
-        assert "openshift-release-artifacts.apps.ci.l2s4.p1.openshiftapps.com/4.12.0-0.nightly-"+ self.year in stream
-        assert "openshift-install-linux-4.12.0-0.nightly-"+ self.year in stream
+        assert "openshift-release-artifacts.apps.ci.l2s4.p1.openshiftapps.com/4.17.0-0.nightly-"+ self.year in stream
+        assert "openshift-install-linux-4.17.0-0.nightly-"+ self.year in stream
 
     def assert_amd_client(self,stream):
         assert "arm64" not in stream
-        assert "openshift-release-artifacts.apps.ci.l2s4.p1.openshiftapps.com/4.12.0-0.nightly-"+ self.year in stream
-        assert "openshift-client-linux-4.12.0-0.nightly-"+ self.year in stream
+        assert "openshift-release-artifacts.apps.ci.l2s4.p1.openshiftapps.com/4.17.0-0.nightly-"+ self.year in stream
+        assert "openshift-client-linux-4.17.0-0.nightly-"+ self.year in stream
 
     def test_manifest_amd(self,mocked_manifest):
         stream = mocked_manifest.latest_releases[self.stream]
@@ -45,13 +45,13 @@ class TestManifest():
 
     def assert_arm_installer(self,stream):
         assert "arm64" in stream
-        assert "openshift-release-artifacts-arm64.apps.ci.l2s4.p1.openshiftapps.com/4.12.0-0.nightly-arm64-"+ self.year in stream
-        assert "openshift-install-linux-amd64-4.12.0-0.nightly-arm64-"+ self.year in stream
+        assert "openshift-release-artifacts-arm64.apps.ci.l2s4.p1.openshiftapps.com/4.17.0-0.nightly-arm64-"+ self.year in stream
+        assert "openshift-install-linux-amd64-4.17.0-0.nightly-arm64-"+ self.year in stream
 
     def assert_arm_client(self,stream):
         assert "arm64" in stream
-        assert "openshift-release-artifacts-arm64.apps.ci.l2s4.p1.openshiftapps.com/4.12.0-0.nightly-arm64-"+self.year in stream
-        assert "openshift-client-linux-amd64-4.12.0-0.nightly-arm64-"+self.year in stream
+        assert "openshift-release-artifacts-arm64.apps.ci.l2s4.p1.openshiftapps.com/4.17.0-0.nightly-arm64-"+self.year in stream
+        assert "openshift-client-linux-amd64-4.17.0-0.nightly-arm64-"+self.year in stream
 
     def test_manifest_arm(self,mocked_manifest):
         stream = mocked_manifest.latest_releases[f"{self.stream}-arm64"]
@@ -63,7 +63,7 @@ class TestManifest():
         hits = 0
         for release in releases:
             release_name = release["release"].get_release_name()
-            if "4.12-aws" in release_name and "arm" not in release_name:
+            if "4.17-aws" in release_name and "arm" not in release_name:
                 self.assert_amd_installer(release["release"].get_latest_release()[self.INSTALL_BINARY])
                 self.assert_amd_client(release["release"].get_latest_release()[self.CLIENT_BINARY])
                 hits += 1

